runtime/util/romclasswalk.c

```diff
@@ -1019,7 +1025,30 @@ allSlotsInOptionalInfoDo(J9ROMClass* romClass, J9ROMClassWalkCallbacks* callback
 	callbacks->sectionCallback(romClass, optionalInfo, (UDATA)cursor - (UDATA)optionalInfo, "optionalInfo", userData);
 }
 
-static void 
+#if defined(J9VM_OPT_METHOD_HANDLE) && !defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+static void
+allSlotsInVarHandleMethodTypeLookupTableDo(J9ROMClass* romClass, J9ROMClassWalkCallbacks* callbacks, void* userData)
```

Comment:
We are iterating through the VarHandle MethodRef CP indices. `allSlotsInVarHandleMethodRefIndexesDo` matches the naming of the adjacent methodref iterators.
```suggestion
allSlotsInVarHandleMethodRefIndexesDo(J9ROMClass* romClass, J9ROMClassWalkCallbacks* callbacks, void* userData)
```